imagersite/imagersite/urls.py

```diff
@@ -14,8 +14,28 @@
     2. Add a URL to urlpatterns:  url(r'^blog/', include(blog_urls))
 """
 from django.conf.urls import include, url
+from django.conf.urls.static import static
 from django.contrib import admin
+from imagersite.views import IndexView, TemplateView
+from django.conf import settings
+from django.contrib.auth.decorators import login_required
 
 urlpatterns = [
+    url(r'^$', IndexView.as_view(), name='home'),
+    url(r'', include('registration.auth_urls')),
+    url(r'^accounts/', include('registration.backends.default.urls')),
     url(r'^admin/', include(admin.site.urls)),
+    url(r'^profile/',
+        login_required(TemplateView.as_view(template_name='profile.html')),
+        name='profile'),
+    url(r'^images/library/', login_required(TemplateView.as_view(
+                                            template_name='library.html')),
+        name='library'),
```

Comment:
As discussed in class, these should be refactored into `urls.py` files in your respective apps, then those should be included back here.